azure-pipelines.yml

```diff
@@ -25,7 +25,7 @@ jobs:
       GIT_USER_EMAIL: $(GIT_USER_EMAIL)
       GIT_USER_NAME: $(GIT_USER_NAME)
       GIT_PASSWORD: $(GIT_PASSWORD)
-  - script: npm run test-staging
+  - script: npm run test-staging && node helpers/integration-tests/index.js
```

Comment:
@uchemukolo can you add this also to line 19 in a separate commit? That way we can test it on Azure:

`  - script: npm test && node helpers/integration-tests/index.js`

We will have to remove it later but that will help us identify if there are any issues with puppeteer on Azure.

Thank you 🙏

Reply:
This has been don and pushed